extraTs/utils/permissions.ts

```diff
@@ -0,0 +1,14 @@
+import { permissions } from '../constants';
+const hasPermission = (moduleName: string, Role: string, PERMISSION_TYPE: string) => {
```

Comment:
take proper arguments name  for .
Role , PERMISSION_TYPE

Reply:
return type is also missing for hasPermission

Reply:
Changes Done.